src/main/java/calculator/StringInput.java

```diff
@@ -9,16 +9,23 @@
 	private static final String CUSTOM_DELIMITER_BOUNDARY = "//(.*)\n(.*)";
 	private static final String EXCEPTION_MESSAGE = "양의 숫자만 입력 가능합니다.";
 
+	private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile(CUSTOM_DELIMITER_BOUNDARY);
+
 	private String input;
 
 	public StringInput(String input) {
-		this.input = input;
+		setInput(input);
```

Comment:
```suggestion
		validate(input);
		this.input = input;
```

setInput() 으로 분리하는 것 보단, validate 로직만 분리해보는건 어떨까요?